src/Microsoft.DotNet.ServiceFabric.ServiceHost/LoadReporting/IServiceLoadReporter.cs

```diff
@@ -0,0 +1,7 @@
+namespace Microsoft.DotNet.ServiceFabric.ServiceHost
+{
+    public interface IServiceLoadReporter
```

Comment:
I think at least this interface ought to define what Load means here and the intended purpose of ReportLoad().

Reply:
I'll add some.